.claude/skills/long-running-task/SKILL.md

```
@@ -0,0 +1,356 @@
+./scripts/bisect_failures --oracle=gcc --system=our-compiler --target=linux-kernel
+# Output: "Files causing failure: fs/ext4/inode.c, mm/page_alloc.c"
+```
+
+---
+
+## 8. Design for Parallelism Explicitly
+
+Parallelism works only if tasks are separable and ownership is enforced.
+
+### Repo-level work queue
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)